net-dialup/freeradius/metadata.xml

```diff
@@ -22,5 +22,8 @@
     <flag name="redis">
       Include support for Redis database
     </flag>
+    <flag name="mongo">
```

Comment:
There's an existing `mongodb` USE flag. Rename it to this. I've committed some profile changes that should allow you to trivially do this.

Reply:
done